bridge/slack/handlers.go

```diff
@@ -308,29 +329,24 @@ func (b *Bslack) skipMessageEvent(ev *slack.MessageEvent) bool {
 		}
 	}
 
-	if !b.GetBool("EditDisable") && ev.SubMessage != nil && ev.SubMessage.ThreadTimestamp != ev.SubMessage.Timestamp {
+	if !b.GetBool(editDisableConfig) && ev.SubMessage != nil && ev.SubMessage.ThreadTimestamp != ev.SubMessage.Timestamp {
 		// it seems ev.SubMessage.Edited == nil when slack unfurls
 		// do not forward these messages #266
 		if ev.SubMessage.Edited == nil {
 			return true
 		}
 	}
 
-	if len(ev.Files) > 0 {
-		for _, f := range ev.Files {
-			// if the file is in the cache and isn't older then a minute, skip it
-			if ts, ok := b.cache.Get("file" + f.ID); ok && time.Since(ts.(time.Time)) < time.Minute {
-				b.Log.Debugf("Not downloading file id %s which we uploaded", f.ID)
-				return true
-			} else {
-				if ts, ok := b.cache.Get("filename" + f.Name); ok && time.Since(ts.(time.Time)) < time.Second*10 {
-					b.Log.Debugf("Not downloading file name %s which we uploaded", f.Name)
-					return true
-				} else {
-					b.Log.Debugf("Not skipping %s %s", f.Name, time.Now().String())
-				}
-			}
+	for _, f := range ev.Files {
+		// if the file is in the cache and isn't older then a minute, skip it
+		if ts, ok := b.cache.Get("file" + f.ID); ok && time.Since(ts.(time.Time)) < time.Minute {
+			b.Log.Debugf("Not downloading file id %s which we uploaded", f.ID)
+			return true
+		} else if ts, ok := b.cache.Get("filename" + f.Name); ok && time.Since(ts.(time.Time)) < time.Minute {
```

Comment:
I've moved this delay back to `time.Minute` instead of `10 * time.Second` as it seemed contradictory with the comment and the other delay.

Reply:
Can revert this back indeed if you want.

Looks like it might be interesting to make this a little bit more solid by rather checking for something like an MD5 of the uploaded file? WDYT?

Reply:
Yes, better to revert this for now. We can't check for md5, because that would mean actually downloading the file. (ofcourse we can but I rather avoid this if possible)
Code is related to https://github.com/42wim/matterbridge/issues/481